cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -400,11 +539,54 @@ def get_rna_bigwig_coverage_upload_summary(
             )
         return upload_summary
 
+    @staticmethod
+    def get_rna_fraser_outrider_upload_summary(
+        rna_dna_collections: list[RNADNACollection],
+    ) -> list[str]:
+        upload_summary: list[str] = []
+        for rna_dna_collection in rna_dna_collections:
+            upload_summary.extend(
+                f"Uploaded fraser and outrider files for sample {rna_dna_collection.dna_sample_name} in case {dna_case}."
+                for dna_case in rna_dna_collection.dna_case_ids
+            )
+        return upload_summary
+
+    @staticmethod
+    def get_rna_genome_build_upload_summary(
+        rna_dna_collections: list[RNADNACollection],
+    ) -> list[str]:
+        upload_summary: list[str] = []
+        for rna_dna_collection in rna_dna_collections:
+            upload_summary.extend(
+                f"Uploaded RNA genome build file for sample {rna_dna_collection.dna_sample_name} in case {dna_case}."
+                for dna_case in rna_dna_collection.dna_case_ids
+            )
+        return upload_summary
+
+    @staticmethod
+    def get_variant_load_summary(
+        rna_dna_collections: list[RNADNACollection],
+    ) -> list[str]:
+        upload_summary: list[str] = []
+        for rna_dna_collection in rna_dna_collections:
+            upload_summary.extend(
+                f"Loaded variant outlier for sample {rna_dna_collection.dna_sample_name} in case {dna_case}."
+                for dna_case in rna_dna_collection.dna_case_ids
+            )
+        return upload_summary
```

Comment:
See previous comment regarding logging

Reply:
agreed, but leaving this for the refactoring